cmp/grid/GridModel.js

```diff
@@ -131,7 +137,12 @@ export class GridModel {
 
     @action
     setGroupBy(field) {
-        const cols = this.columns;
+        const cols = this.columns,
+            groupCol = find(cols, {field});
+
+        // If we have an invalid groupBy field do not set
+        // Let a falsey field through to mean 'no grouping'
+        if (field && !groupCol) return;
```

Comment:
Is this what we want in terms of behavior? Seems nice to have a way to reset the grouping, but I could also see 'false' meaning "don't change the grouping".

The latter might be helpful if we ever decide to save grouping as part of state. In which case, if we didn't have a groupBy field saved we'd still get the default grouping if we call this. We could also just NOT call this from the stateModel when loading state if we have no groupBy saved. (Perhaps I'm getting ahead of myself?)